_posts/2019-10-03-facilitation-dont-lose-before-you-start.markdown

```diff
@@ -0,0 +1,99 @@
+# Facilitation: Don't Lose Before You Start
+
+So you want to facilitate a workshop for a team? Maybe you're external to the team and have been asked to help by facilitating. Awesome, it'll be fun. Just show up at the time and place annnnnnnd you've already lost.
+
+> You better think (think)
+
+> Think about what you're trying to do
+
+-- [Aretha Franklin](https://en.wikipedia.org/wiki/Aretha_Franklin)
+
+## Ask The Right Question
+
+The workshop should be answering a question. That gives all of the participants a clear vision of what they're trying to achieve. "What is preventing the team from moving faster?" is a very different workshop than "How do we improve our deployment process?" which is very different from "How do we fully automate our deployment process?"
+
+## Get to Know Your Stakeholders
+
+Someone had to feel strongly enough about this to put a multi-hour block of time on the team's calendar. This is the First Stakeholder. Start with them. You need to understand what they want out of this workshop. What outcomes will make them call this a success? Help them understand their role in the workshop: getting the right people in the room, taking a step back, and letting the team be awesome.
+
+Hopefully the First Stakeholder has conveyed their vision of success to the team. Validate that assumption. Talk to the rest of the team. Understand what they want to get out of the workshop. It's better to discover any differences ahead of time rather than in the workshop itself.
+
+Take advantage of those conversations with the team members to hear their perspectives, their pains, their goals for the workshop. Is it scheduled at the same time as their kid's play at school but they're forced to attend? That's good to know. Find out who on the team is quiet, who interrupts, who isn't afraid to speak their mind. Find out who on the team likes the status quo and [doesn't feel change is necessary](https://hbr.org/2013/07/hidden-danger-of-being-risk-averse).
+
+Sometimes it's necessary to have a second sit-down with the First Stakeholder - if what you're learning isn't making sense, or there are incompatible goals, if some constraints can't possibly be met, etc. Help them understand what you've learned. Be the team's advocate and share the team's perspective.
```

Comment:
This may be nitpicky but the first sentence in this paragraph wasn't the smoothest for me to read the first time around.  I'm not sure if multiple sentences is the answer but thought I'd mention it.

Reply:
Is the new wording any clearer?

Reply:
That reads easier for me, hopefully you're also happy with it compared to the original wording.  The new wording also makes this section sound more cohesive somehow.  I think it's because it matches the pattern of the first paragraph (if... then ...).